frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -76,14 +79,17 @@ const DevPortfolioForm: React.FC = () => {
       ? devPortfolio.lateDeadline
       : devPortfolio?.deadline;
 
-    if (!isTpm && (openedEmpty || reviewedEmpty)) {
-      Emitters.generalError.emit({
-        headerMsg: 'No opened or reviewed PR url submitted',
-        contentMsg: 'Please paste a link to a opened and reviewed PR!'
-      });
+    if (!isTpm && otherEmpty) {
```

Comment:
Rewrite the condition as `!isTpm && otherEmpty && (openedEmpty || reviewedEmpty)` and the delete the inner if-statement. If you try it out, this logic doesn't allow you to submit regular Dev Portfolios without otherPRs.

Reply:
Thanks for pointing this out! I had a misunderstanding that if otherPRs is empty, then both an opened PR and a reviewed PR had to be submitted.